### PR TITLE
Campaign finance: new data schema

### PR DESCRIPTION
**Pull request by ybyzek**
## Problem

Related to new recipe: https://github.com/confluentinc/kafka-tutorials/pull/1220

##### Does this solution apply anywhere else?
- [ ] yes
- [ ] no

##### Testing done:
- [ ] Unit tests
- [ ] Integration tests
- [ ] System tests
- [ ] Manual tests
- [x] Validated with the recipe at https://github.com/confluentinc/kafka-tutorials/pull/1220

### CODE REVIEW

File: connector-test.sh [hunk 1]
```diff
@@ -10,7 +10,8 @@ sleep 10
 
 confluent local services connect status
 
-connectors="credit_cards stores transactions purchases"
+#connectors="credit_cards stores transactions purchases"
```
Comment (shaikzakiriitm) [inline connector-test.sh:13]: you may add `campaign_finance` in the same list of connectors at the end.

File: config/connector_campaign_finance.config [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "datagen-campaign_finance",
+  "config": {
+    "connector.class": "io.confluent.kafka.connect.datagen.DatagenConnector",
+    "kafka.topic": "campaign_finance",
+    "quickstart": "campaign_finance",
+    "key.converter": "org.apache.kafka.connect.storage.StringConverter",
```
Comment (shaikzakiriitm) [inline config/connector_campaign_finance.config:7]: need a `schema.keyfield=cand.id` here?

File: src/main/resources/campaign_finance.avro [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+                {"name": "time", "type": {
+                  "type": "long",
+                  "arg.properties": {
+                        "range": {
+                            "min": 1587715775521,
+                            "max": 1619273364600
+                        }
+                  }
+                }},
+                {"name": "cand_id", "type": {
```
Comment (shaikzakiriitm) [inline src/main/resources/campaign_finance.avro:15]: what does `cand_id` stand for? Should we rename it to something intuitive?

**Review by shaikzakiriitm — COMMENTED** [review]
looks good. few minor comments.

File: src/main/resources/campaign_finance.avro (hunk 3 above)
Comment (ybyzek) [reply src/main/resources/campaign_finance.avro:15]: 👍 and done in latest commit.

File: connector-test.sh (hunk 1 above)
Comment (ybyzek) [reply connector-test.sh:13]: This can be changed depending on what is being tested.

File: config/connector_campaign_finance.config (hunk 2 above)
Comment (shaikzakiriitm) [reply config/connector_campaign_finance.config:7]: sure. Was asking as the other pr had it mentioned explicitly https://github.com/confluentinc/kafka-connect-datagen/pull/106

**Review by shaikzakiriitm — COMMENTED** [review]
lgtm, can you please take a look at the alerts (check failures) in the pr, mostly line 83 and line 90 in pom.xml file need to be updated to use https. This will help us get a green build on jenkins.

**Review by shaikzakiriitm — APPROVED** [review]
lgtm, but please get a green build before merging. Thanks @ybyzek

**Pull request merged**